cmd/kubeadm/app/phases/addons/proxy/manifests.go

```diff
@@ -75,6 +75,8 @@ spec:
         command:
         - /usr/local/bin/kube-proxy
         - --kubeconfig=/var/lib/kube-proxy/kubeconfig.conf
+        {{ .KubeProxyMode}}
+        {{ if .KubeProxyMode }}- --feature-gates=AllAlpha=true,SupportIPVSProxyMode=true{{ end }}
```

Comment:
should kubeadm enable all alpha features?

Reply:
Sorry, my mistake!